assets/app/views/site/edit.js

```diff
@@ -3,7 +3,7 @@ var fs = require('fs');
 var Backbone = require('backbone');
 var _ = require('underscore');
 
-var templateHtml = fs.readFileSync(__dirname + '/../templates/SiteEditTemplate.html').toString();
+var templateHtml = fs.readFileSync(__dirname + '/../../templates/site/edit.html').toString();
```

Comment:
Why aren't you just using "require" for these templates? I was pretty sure that was possible in some sort of way.

I would say its cleaner and better structure then using readFileSync.